src/WebJobs.Script/Description/FunctionGenerator.cs

```diff
@@ -7,6 +7,7 @@
 using System.Linq;
```

Comment:
Updated all local references to use the `LocalBuilder` instances
Changed `Wait` call to use `GetAwaiter().GetResult();`